setup.py

```diff
@@ -119,6 +119,7 @@
             'ansible': ANSIBLE_REQUIRES,
         },
         include_package_data=True,
+        package_data={'pyinfra': ['py.typed', 'api/*.pyi']},
```

Comment:
I believe this should be in `MANIFEST.in` rather than here, they'll override the manifest file currently!

Reply:
Ah, I've been using Poetry myself lately, so I wasn't really sure how to handle that. Easy enough to change.